package.json

```diff
@@ -22,6 +22,8 @@
     "backbone.paginator": "~2.0.3",
     "gulp-shell": "^0.5.2",
     "jquery": "~2.2.0",
+    "moment": "^2.15.1",
+    "moment-timezone": "~0.5.5",
```

Comment:
This is a good Q - I think `^` for moment and `~` for moment-timezone is the right choice here because ^ will pick up all changes to the minor version (ie, everything up to 3.0.0) but ~ will limit changes to the patch version (ie, it will pick up 0.5.6, but it will not pick up 0.6) - because the API might have changed there.